market/connector/uipath/meta.json

```diff
@@ -6,22 +6,16 @@
 		{
 			"id": "project-resource",
 			"data": {
-				"resources": [
-					{
-						"source": "demo/uiPathDemo/src/com/uipath/orchestrator/feature/TenantHeaderFeature.java",
```

Comment:
if you don't copy those any longer to the project then I think you can also delete the resources itself?